package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "install": "for d in packages/*/ ; do (cd \"$d\" && npm install); done",
     "flow": "flow",
-    "jest": "jest",
-    "test": "flow && jest",
+    "jest": "STORYBOOK_INLINE_APHRODITE=1 jest",
```

Comment:
I think we need to add this to `travis.yaml`, no?  https://docs.travis-ci.com/user/environment-variables/#Defining-public-variables-in-.travis.yml